### PR TITLE
[IMP] accounting: note about outstanding accounts

### PR DESCRIPTION
**Pull request by Donapi**
https://www.odoo.com/web#id=2868642&cids=1&model=project.task&view_type=form
task-2868642

### CODE REVIEW

**Review by xpl-odoo — APPROVED** [review]
I would have phrased the last part of the sentence this way - but it's fine either way :)
_the invoice or bill's status is directly set to "paid"_

File: content/applications/finance/accounting/bank/setup/outstanding_accounts.rst [hunk 1]
```diff
@@ -44,3 +44,7 @@ Payments` tab. To display the outstanding accounts column, click on the toggle b
 .. note::
    If you do not specify an outstanding payments account or an outstanding receipts account for a
    specific journal, Odoo uses the default outstanding accounts.
+
+.. tip::
+   If your main bank account is added as an outstanding receipts or payments account, when a payment
+   is registered, the invoice or bill's status is directly set to "paid".
```
Comment (jcs-odoo) [inline content/applications/finance/accounting/bank/setup/outstanding_accounts.rst:50]: guilabel :)

**Review by jcs-odoo — CHANGES_REQUESTED** [review]

**Comment by jcs-odoo** [comment]
@Donapi Isn't this true in 14.0 as well?

**Comment by jcs-odoo** [comment]
@robodoo r+

**Review by jcs-odoo — APPROVED** [review]

**Pull request closed without merging**